.github/actions/sparkle-appcast/action.yaml

```diff
@@ -136,9 +136,7 @@ runs:
         if (( ${+RUNNER_DEBUG} )) setopt XTRACE
 
         git tag -l --format='%(contents)' ${GITHUB_REF_NAME} \
-          | tr '\n' '\\n' \
-          | sed 's/-----BEGIN SSH SIGNATURE-----.*-----END SSH SIGNATURE-----//g' \
-          | tr '\\n' '\n' > notes.rst
+          | sed '/-----BEGIN SSH SIGNATURE-----/,/-----END SSH SIGNATURE-----/d' > notes.rst
 
         sed -i '' '2i\'$'\n''###################################################' notes.rst
```

Comment:
Instead of modifying the output of `git tag`, I'd like to suggest using the format options `--format='%(contents:subject)` and `--format='%(contents:body)'` to retrieve the subject and body separately.
```suggest
git tag -l --format='%(contents:subject)' ${GITHUB_REF_NAME} > notes.rst
echo '###################################################' >> notes.rst
echo '' >> notes.rst
git tag -l --format='%(contents:body)' ${GITHUB_REF_NAME} >> notes.rst
```

Reply:
I agree, otherwise PGP signed tags will have that signature come up in the release notes as well. Better to explicitly extract only the fields we need/want then extract everything that can have unexpected fields or data.

Reply:
I've used the multiline form suggested above, as I think it's more readable/decipherable.